parallel_process_data.py

```diff
@@ -0,0 +1,600 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""Process wind data for adjacent years and save processed data to a NetCDF file. Settings, such as target file name,
+are imported from config.py.
+
+Example::
+
+    $ python process_data.py
+
+"""
+from netCDF4 import Dataset
```

Comment:
yes it does, to write the output - do you think it would be better to change all of that to xarray, too? 

That would mean to change the handling of the res dictionary and I think also the whole definition of the output_variables dimensions and so on to using an xarray dataset 
then this dataset would still be written to a netcdf4 file 

so I think the only merit would be a possibly prettier coding, as the res dictionary could maybe be modified to be an xarray dataset, then just write that...? I'm still getting into the use of xarray in this case, but I think that would be the workflow